### DISCUSSION TITLE
How to get to public content after enrolling

### DISCUSSION THREAD

**Discussion by AndrewFalanga** [GitbookIO/community#194] [0. General bugs and questions]
Hi,

So I was really liking the books on the EDK 2 development environment that were in gitbook. Since I'm on GitHub, I decided I'd try gitbook myself. However, now, every time I try to get to documentation which I could reach before I'm unable to do so. I get unauthorized errors and so forth.

Why is it that after signing up I'm unable to view this content? What must I do?

Thanks,
Andy

**Comment by aikoii** [comment]
Hi @AndrewFalanga,

Could you email us at [support@gitbook.com](mailto:support@gitbook.com) and let us know what email are you using to access that space 🙏? It sounds like you might be missing permission to view those docs. 

Thanks!

> Disclaimer: Please note that this is not our official support channel. If you encounter any bugs or issues, we strongly encourage you to contact [support@gitbook.com](mailto:support@gitbook.com).
